app/templates/flat-page.hbs

```diff
@@ -20,21 +20,21 @@
               <div class="share-tools clearfix">
                 <ul class="sharelinks clearfix">
                   <li class="fa-hover facebook">
-                    <a title="Share on Facebook" class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="facebook" data-story-title="About+WQXR" data-show-name="About+WQXR">
+                    <a class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u={{url-encode absoluteURL}}" type="Facebook" data-action="Share Flatpage" data-label={{concat flatPage.title ' | Facebook'}}>
```

Comment:
This is a kinda nontraditional way to use the `type` attribute, it's usually for the media type of the linked document. What's your intent here?

Reply:
ahh yes that was a mistake. I was matching what they did for the story template share buttons, but didn't realize type was just a variable and not an attribute. I'll change that.

Reply:
what would be a better tag for that sort of info? just go back to title?

Reply:
What's the intention, describing the link for accessibility? If so, `aria-label` is best if the link doesn't actually wrap text, like in your case.

Reply:
https://a11yproject.com/posts/title-attributes/

Reply:
yes, that was the intention. I'll make it an aria-label